_includes/footer.html

```diff
@@ -1,6 +1,8 @@
 </main>
 <footer class="mk-main-footer">
-  <p>&copy; 2019 metal3.io <a href="/privacy-statement.html">Privacy Statement</a></p>
+  <p>We are a <a href="https://cncf.io/">Cloud Native Computing Foundation</a> sandbox project.</p>
+  <p><img src="/assets/images/cncf-color.png"/></p>
+  <p>&copy; 2020 The Metal³ Contributors - <a href="/privacy-statement.html">Privacy Statement</a></p>
```

Comment:
Sorry to insist, but if I understand the [guidelines](https://github.com/cncf/foundation/blob/master/copyright-notices.md#copyright-notices) correctly they recommend to use the word _Copyright_ instead of the (c) symbol.

While we are at it, we could make the year auto-updated like this:
`Copyright {{ 'now' | date: "%Y" }} The Metal³ Contributors`

Reply:
I appreciate the tips!

Reply:
done